loggingutils/grpc.go

```diff
@@ -0,0 +1,15 @@
+package loggingutils
+
+import (
+	"context"
+	"github.com/solo-io/go-utils/contextutils"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+)
+
+func LoggerInterceptor(logger *zap.SugaredLogger) grpc.UnaryServerInterceptor {
```

Comment:
Since there are other types of interceptors we might want to implement in the future (`UnaryClientInterceptor`, `StreamServerInterceptor`, `StreamClientInterceptor`), can we use a more specific name for this constructor? Maybe `GrpcUnaryServerLoggerInterceptor` (even though that's a mouthful...)?

Reply:
good point